pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -478,8 +477,20 @@ class Build {
                                     }
                                 }
                                 context.println "Use Test_Job_Auto_Gen to generate AQA test job with parameters: ${updatedParams}"
-                                context.catchError {
-                                    context.build job: 'Test_Job_Auto_Gen', propagate: false, parameters: updatedParams
+                                int x = 3
```

Comment:
x --> retry_attempts?

Reply:
It's the number of attempts in total, so I've renamed the variable appropriately.